app/bundles/FormBundle/Translations/en_US/messages.ini

```diff
@@ -34,6 +34,7 @@ mautic.form.action.sendemail.dragfield="Click to insert field"
 mautic.form.action.sendemail.message="Message"
 mautic.form.action.sendemail.subject="Subject"
 mautic.form.action.sendemail.subject.default="Your form results"
+mautic.form.action.sendemail.file.attached="Attach file in email"
```

Comment:
Could be Attach files in email or different workding. Files from form as attachment